src/js/aos.js

```diff
@@ -38,7 +38,9 @@ let options = {
   duration: 400,
   disable: false,
   once: false,
-  startEvent: 'DOMContentLoaded'
+  startEvent: 'DOMContentLoaded',
+  root: null,
+  scrollContainer: null
```

Comment:
This is a duplicate of https://github.com/michalsnik/aos/pull/393. The exact same feature is provided in the later pull request as `container`.